clusters/app.ci/registry-access/node-observability-operator/admin_node_observability_operator.yaml

```diff
@@ -0,0 +1,62 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: node-observability-operator
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: image-puller
+    namespace: node-observability-operator
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:unauthenticated
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: system:image-puller
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: node-observability-operator-dev
+  users:
```

Comment:
Please user Rover group for RBACs.
https://docs.ci.openshift.org/docs/how-tos/rbac/

Reply:
Created a Rover group, should I do any configuration for `app.ci` cluster or it'll be synced automatically?